src/mixed-reality/azext_mixed_reality/custom.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+
+def create_spatial_anchors_account(client, resource_group_name, spatial_anchors_account_name, location=None, tags=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
You don't need to define all of this in one line. If the line is too long, wrap around. For example this.
```suggestion
def create_spatial_anchors_account(client, resource_group_name, 
     spatial_anchors_account_name, location=None, tags=None, 
     custom_headers=None, raw=False, **operation_config):
```